example/spec/parallel/ingestGranule/IngestGranuleWithDuplicateHandlingErrorSpec.js

```diff
@@ -25,19 +26,22 @@ const { loadConfig } = require('../../helpers/testUtils');
 describe('The IngestGranuleCatchDuplicateErrorTest workflow with DuplicateHandling = "error" and a granule re-ingested', () => {
   let beforeAllFailed = false;
   let collection;
+  let config;
   let firstIngestGranuleRule;
+  let firstIngestGranuleExecutionArn;
```

Comment:
Yeah, I should have updated these as I went. Instead I just did whichever format was in the test. Here's an update to use numerals instead of `first`/`second`. 

https://github.com/nasa/cumulus/pull/2252/commits/3341ede4e9d6bb4a676754070fa558a4e2ff22c3